js/browzine.js

```diff
@@ -7,44 +7,62 @@
       $sceDelegateProvider.resourceUrlWhitelist(urlWhitelist);
   }]);
 
-// Add Browzine Links
-  app.controller('prmBriefResultAfterController', function($scope, $http) { 
+// Add Article In Context & Browzine Links
+  app.controller('prmSearchResultAvailabilityLineAfterController', function($scope, $http) { 
     var vm = this;
-    if (vm.parentCtrl.item.pnx.addata.doi)  {
-          vm.doi = vm.parentCtrl.item.pnx.addata.doi[0] || '';
-          var url = "https://yourserver.edu/primo/browzine/browzineArticleInContext?DOI=" + vm.doi;
-          $http.jsonp(url, {jsonpCallbackParam: 'callback'}).then(function(response) {
-          $scope.article = response.data;
-          }, function(error){
-                 console.log(error); //
+		if (vm.parentCtrl.result.pnx.addata.doi && vm.parentCtrl.result.pnx.display.type[0] == 'article')	{
```

Comment:
Little formatting nit here, but the (probably unintentional?) whitespace changes here can tend to mess up diffs and make reviews a little longer to get through.

I thought I'd mention something that's been useful on our team: the `.editorconfig` file, as described at http://editorconfig.org

You put it in the root directory of your project, and when opening that directory in a text editor, many text editors recognize the preferences there and will respect them, independent of their own preferred settings.  It'll help changes like this here - swapping in tabs for spaces - from unintentionally happening.

Hopefully this wasn't too much of a bike shed comment 😊  🚲